modules/migration-debugging-velero-admin-webhooks.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+[id="migration-debugging-velero-admin-webhooks_{context}"]
+= Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore, so you might need to use an additional Velero plug-in or make changes to how you restore the workload.
+
+Typically workloads with admission webhooks require you to create a resource of a specific kind first.
+
+Documented restore workarounds:
+
+* link:https://github.com/knative/serving/issues/10926#issuecomment-886434177[Knative] requires you to restore the top level `Service` resource first by entering the following command:
++
+[source,terminal]
+----
+$ velero restore <restoreName> --from-backup=<backupName> --include-resources \
```

Comment:
You could update the replaceable value references so that they are lowercase and use underscores to separate the words. For example:

* `<restore_name>`
* `<backup_name>`

For more information, see [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) in the OpenShift documentation guidelines.